test/pytest/test_metrics_kf.py

```diff
@@ -11,13 +11,16 @@
 import test_utils
 
 NUM_STARTUP_CFG = 0
-REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../")
-snapshot_file = os.path.join(REPO_ROOT,"test/config_kf.properties")
+REPO_ROOT = os.path.normpath(
+    os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../")
```

Comment:
@DPeled Hi, I've seen the resolved conversation about this topic above but could you elaborate why e.g. [this line](https://github.com/pytorch/serve/blob/3c02fb67e12c82cb3231226a3a947ed93b519f4f/examples/text_to_speech_synthesizer/create_mar.py#L16) needs to be changes  from join('A','B/C') into join('A','B','C') but here we do not need to to join('A', '..','..')

Reply:
Of course!

Apparently the underlying code where the real accessing to that path is happened, Windows can understand that type of paths (like `C:\PATH\TO\SUBDIR\../../` which the join produce).

But, moreover, to eliminate that strange looking path, the `os.path.normpath` function takes this path and normalize it to look just like any traditional path in the current OS.